### ISSUE TITLE
Feature request: Add text value in status 'summon'

### ISSUE THREAD

**Issue by Hursev** [michaelnpsp/Grid2#248]
Hello,
Can you add a text value for the 'summon' status values?
The code can be something like this in StatusSummon.lua

```Lua
local textDeclined = L["Declined"]
local textAccepted = L["Accepted"]
local textPending = L["Pending"]
function status:GetText(unit)
	local state, text = IncomingSummonStatus(unit)
	if state == 3 then -- declined(3)
		color = textDeclined
	elseif state == 2 then -- accepted(2)
		text = textAccepted
	elseif state == 1 then -- pending(1)
		text = textPending
	else -- none. We should never hit this line.
		text = ""
	end
	return text
end
```
Plus the localization.

Thank you,
Hursev

**Comment by michaelnpsp** [comment]
Implemented in  version 2.8.39-beta

**Comment by Hursev** [comment]
Thank you!
As far as I tested, it works as intended in the 2.8.39-beta.
